src/templates/Navigator/MainNavigatorComponent/Readme.md

```diff
@@ -0,0 +1,5 @@
+MainNavigatorComponent Example:
+
+    <MainNavigatorComponent>
```

Comment:
just a comment, that we can probably avoid adding `Component` to the names of our components. they're all components, so this suffix doesn't add any new information